src/pages/index.tsx

```diff
@@ -21,17 +21,31 @@ import ShopLookupSkeleton from "../components/lookup/ShopLookupSkeleton";
 
 const MAX_CHIP_MOBILE = 0;
 const MAX_CHIP = 3;
-const filterByPartType = ["OEM", "AFTERMARKET"];
-const filterByPartCondition = ["NEW", "USED"];
+const partTypes: { key: string; value: string }[] = [
+  { key: "No Filter Type", value: "No Filter" },
+  { key: "OEM", value: "OEM Only" },
+  { key: "AFTERMARKET", value: "Aftermarket Only" },
+  { key: "OEM & AFTERMARKET", value: "OEM & Aftermarket" },
+];
+const partConditions = [
+  { key: "No Filter Condition", value: "No Filter" },
+  { key: "NEW", value: "Only New Parts" },
+  { key: "USED", value: "Only Used Parts" },
+  { key: "NEW & USED", value: "New & Used Parts" },
+];
```

Comment:
You could mark these as `const`, so you don't need to use `!` everywhere to override the type. E.g., `partTypes[0]!.key` becomes `partTypes[0].key`.

```suggestion
const partTypes = [
  { key: "No Filter Type", value: "No Filter" },
  { key: "OEM", value: "OEM Only" },
  { key: "AFTERMARKET", value: "Aftermarket Only" },
  { key: "OEM & AFTERMARKET", value: "OEM & Aftermarket" },
] as const;
const partConditions = [
  { key: "No Filter Condition", value: "No Filter" },
  { key: "NEW", value: "Only New Parts" },
  { key: "USED", value: "Only Used Parts" },
  { key: "NEW & USED", value: "New & Used Parts" },
] as const;
```